pages/common/emacs.md

```diff
@@ -0,0 +1,11 @@
+# emacs
+
+> A command-line text editor (and operating system)
+
+- Exit emacs
+
+`CTRL`+`x`, `CTRL`+`c`
```

Comment:
Here the 2/3 LGTM policy would have helped; I would argue that this notation is an emacs idiom and we should probably use something a little more generic, aimed at beginners. You made this point yourself recently in another PR regarding `cr` vs. `enter`.